Linux/User and File Management/default-permissions/understanding-umask.md

```diff
@@ -10,17 +10,13 @@ type: normal
 category: must-know
 
 tags:
-
   - linux
-
   - permissions
-
   - umask
-
   - terminal
-
   - files
 
+
```

Comment:
it might be that we used 🔮  instead of 🐉  in some places?

Reply:
I checked the revamp file and those insights/workouts have no tags.